app/controllers/documents_controller.rb

```diff
@@ -2,6 +2,11 @@
 
 class DocumentsController < ApplicationController
   def index
+    if filter_params[:filters].empty?
```

Comment:
Would an alternative be to default the param in the `EditionFilter`? My thinking here is that `documents_path` offers a 'default view' for the user, which might be equivalent to a specific search. I understand it might be a bit weird being able to get the same data from two different URLs, but I can't see any practical downside to this, and the concept of the unadorned `documents_path` as a 'default view' or 'dashboard' sounds pretty intuitive to me.